### PR TITLE
Included FUN values in the page

### PR DESCRIPTION
**Pull request by tarikcoskun**
There was some space at the bottom and it was looking bad so I came up with the idea of filling it up with FUN values, it now looks like so: 
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -312,6 +312,60 @@ <h1>SAVE Data (file0)</h1>
...
+                            <td>2-39</td>
+                            <td>Wrong number</td>
+                        </tr>
+                        <tr>
+                            <td>40-45</td>
+                            <td>Sans' call</td>
+                        </tr>
+                        <tr>
+                            <td>46-50</td>
+                            <td>Alpyhs' call</td>
```
Comment (KockaAdmiralac) [inline index.html]: `Alphys'` or `Alphys's`, but not `Alpyhs'`.

File: index.html [hunk 2]
```diff
@@ -312,6 +312,60 @@ <h1>SAVE Data (file0)</h1>
...
+                        <tr>
+                            <td>66</td>
+                            <td>Fake hallway</td>
+                        </tr>
+                        <tr>
+                            <td>80-89</td>
+                            <td>Clam girl</td>
+                        </tr>
+                        <tr>
+                            <td>91-100</td>
```
Comment (KockaAdmiralac) [inline index.html]: The `gml_Object_obj_mkid_goner_Create_0` script says:
```cpp
        if((global.flag[5] >= 90) && file_exists("undertale.ini")) {
            // ...
```
so I believe the right fun value is 91 and above, not 91-100.

File: index.html (hunk 1 above)
Comment (tarikcoskun) [reply index.html]: i can never type it correctly

**Comment by crumblingstatue** [comment]
Hey, that's pretty cool, thanks!

**Comment by crumblingstatue** [comment]
I'm letting you 2 have access to this repository, because I haven't been interested in this project for a long time.
Enjoy responsibly!

**Pull request merged**